src/services/rolls/constants/index.ts

```diff
@@ -46,6 +46,8 @@ export interface Bounty {
   reward: number
   rewardDenomination: RewardDenomination
   createdAt: string
+  startedAt?: string
```

Comment:
re: startDate needs to be added, but startedAt does provide relevant information as well obviously. so i'd include both?